src/components/scenes/PasswordLoginScene.tsx

```diff
@@ -52,6 +52,7 @@ const MAX_DISPLAYED_LOCAL_USERS = 5
 
 export interface PasswordLoginParams {
   username: string
+  createAccountType?: 'full' | 'light'
```

Comment:
Should declare an 

`export type CreateAccountType = 'full' | 'light'` for use by the GUI